docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc

```diff
@@ -32,6 +32,18 @@ may experience issues updating. To resolve this problem:
 
 . In a terminal window, run the following `cURL` request, providing your {kib} superuser credentials to reset the {ecloud} agent policy.
 
+** On {kib} versions 8.11 and higher, run:
++
+[source,shell]
+----
+curl -u <username>:<password> --request POST \
+  --url <kibana_url>/internal/fleet/reset_preconfigured_agent_policies/policy-elastic-agent-on-cloud \
+  --header 'content-type: application/json' \
+  --header 'kbn-xsrf: xyz' \
+  --header 'elastic-api-version: 1'
+----
+
+** On {kib} versions below 8.11, run:
```

Comment:
```suggestion
** On {kib} versions earlier than 8.11, run:
```

Reply:
Below implies physical positioning. In this case, it's more of a timeline concept.